README.md

```diff
@@ -1,28 +1,29 @@
+# ORise's ROS Workspace
+
 ![docker-deploy-foxy](https://github.com/open-br/ros_ws/workflows/docker-deploy-foxy/badge.svg?branch=master)
 ![docker-deploy-dashing](https://github.com/open-br/ros_ws/workflows/docker-deploy-dashing/badge.svg?branch=master)
 ![docker-build](https://github.com/open-br/ros_ws/workflows/docker-build/badge.svg?branch=master)
 
-# My ROS Workspace
-
 This repository provides simple tools to develop and test ROS and ROS2 packages using Docker.
 
-## Build Images
+## Develop within a Container
 
-Dockerfile defines multi-stage build with the targets:
-1. `test-build`: the minimum needed to download and build a ROS package
-2. `devel-build`: `test-build` + `ros-$ROS_DISTRO-ros-base` packages
-
-The script `build-images.sh` automates the creation of the images given the target ROS distro name. For example:
+The script `run_devel.sh` creates a devel container (or start/attach to an existing one) given the target ROS distro. For example, the command:
 ```console
-./build-images.sh -d dashing
+./run-devel.sh -d melodic
 ```
-Creates the develop image `ros-dashing:devel`. If you want the test image `ros-dashing:test` instead, you may pass the `--test` argument:
+will attach to the container `ros-melodic-devel`.
+
+### Running sudo commands
```

Comment:
We may add the sudoers privilege, if so this part would be deprecated.

Reply:
Removed in 417d544